apps/fabric-website/src/SiteDefinition/SiteDefinition.pages/Controls/web.tsx

```diff
@@ -251,6 +257,24 @@ function _generatePage(
   };
 }
 
+function _loadReferences(): INavPage[] {
+  const requireContext = require.context('@uifabric/api-docs/lib/pages/references', false, /\w+\.page\.json$/, 'lazy');
```

Comment:
Cool!

Reply:
Yeah I had no idea about this either until Ken told me about it recently!